### ISSUE TITLE
Request: Option to local boot

### ISSUE THREAD

**Issue by dholt** [google/netboot#58]
If not already possible, it'd be nice to have an option to boot the local disk, i.e. if the API returns 404, send the PXE arg "LOCALBOOT 0" to bypass waiting until the client PXE ROM times out. Happy to try and work on it but thought I'd add a feature request.

**Comment by danderson** [comment]
Yeah, that sounds like a good idea. What does the PXE spec say about that response, does it just expedite exiting the PXE ROM? If it does something other than make the timeout happen faster, it would have to be optional, because it might mess with existing workflows.

I don't really have time to hack on this myself, but I'd welcome PRs!
